ui/app.py

```diff
@@ -74,12 +74,31 @@ def dalle2(text_input):
     else:
         st.image(DEFAULT_IMG_URL, width=500)
 
+        
+def get_semantic_search_results(question):
+    whoami = "I am a highly intelligent question answering bot against lowering CO2 emission. If you comment about your plan for popular eco-friendly solutions, I will answer with the side effects of those."
+    start_sequence = "\nA:"
+    restart_sequence = "\n\nQ: "
+    response = openai.Completion.create(
+      model="text-davinci-002",
+      prompt=whoami + "\n\nQ: "+ question +"\nA:",
```

Comment:
nit: we can either reuse `start_sequence` and `restart_sequence` for `prompt`, or remove those parameters?

Reply:
I'm removing those two, you're right, they're not used anymore.